devtools/server/actors/replay/network-helpers.jsm

```diff
@@ -61,7 +61,7 @@ function getChannelResponseData(channel, fromCache) {
     responseStatus: channel.responseStatus,
     responseStatusText: channel.responseStatusText,
     responseFromCache: !!fromCache,
```

Comment:
I have not thought about this really, but off the cuff I wonder if we should consider adding a `responseSource: "network" | "cache" | "serviceworker"` or something.

Reply:
see my comment below, but I'm starting to suspect that `responseFromCache` and `responseFromServiceWorker` should be two distinct boolean flags. Which I guess agrees more with the current design anyways. Hopefully I'll have a better understanding after reading some more code.

Reply:
Sounds good. Seems like we should add `responseFromServiceWorker: !!fromServiceWorker` here then so the frontend can indicate in some way that a request came from a serviceworker instead of a server. We'll also want to update the protocol definitions, but either can land first.